seneca.js

```diff
@@ -475,7 +495,7 @@ function make_seneca (initial_options) {
               subfunc.call(self, args, result)
             }
             catch (ex) {
-              // TODO: not really satisfactory
+              // TOO: not really satisfactory
```

Comment:
@rjrodger Was this intentional? If it wasn't, next time you search for "TODOs" you could miss this.